packages/localize-tools/src/program-analysis.ts

```diff
@@ -684,7 +696,6 @@ function contentEqual(
     if (typeof b !== 'object') {
```

Comment:
Could simplify this now to `if (typeof a === 'object' && typeof b !== 'object')`